dask/array/tests/test_masked.py

```diff
@@ -138,7 +138,7 @@ def test_mixed_random(func):
     dd = func(d)
     ss = func(s)
 
-    assert_eq(dd, ss)
+    assert_eq(dd, ss, check_meta=False)
```

Comment:
If masked arrays aren't handling meta well can I ask you to raise an issue?  I can ping some people that I think are motivated to keep it up to date, and maybe they can help.

Reply:
It's not the case they're not handled well, but that we're indeed comparing a masked array to a regular one, which means types really don't match, so we have to purposely ignore comparing `_meta`. All other tests (where we compared two masked arrays) work well. Do you have a different suggestion here? Otherwise, I think these two cases are handled properly as they are now.